### PR TITLE
Bug 1131521 - Running r-wrapper.js on node.js

### PR DESCRIPTION
**Pull request by rickychien**
https://bugzilla.mozilla.org/show_bug.cgi?id=1131521

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d5bd33ecd16a769e73f23556a786dba8ee8687b6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a6a7718cd6ac785b8493f99fe36fa67de513e8c7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f2defa4bcd95cf1ae534c06f2a5ad0a30518f63

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ee4f91c03311208fe410daa939093f7023d0e76

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0de6b0fe4894cde9977151f6ba6acdcce786445e

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "node": "^4 || ^5"
   },
   "devDependencies": {
+    "SMS": "file:./apps/sms",
```
Comment (rickychien) [inline package.json]: When we do ```npm install xxx --save```, npm will rearrange dependencies automatically. So I prefer to keep this change in this patch.

File: apps/bluetooth/build/build.js [hunk 2]
```diff
@@ -1,25 +1,36 @@
...
-exports.execute = function(options) {
-  (new BluetoothAppBuilder()).execute(options);
+  return build.then(function() {
+    utils.log(appName, 'require.js optimize done');
+  }).catch(function(err) {
+    utils.log(appName, 'require.js optimize failed');
+    throw err;
+  });
 };
+
```
Comment (scottwu) [inline apps/bluetooth/build/build.js]: nit: remove extra line

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=701792d11de94fe95c2e5c828c3d5f2b97059b97

**Pull request merged**